MapboxDirections/MBRouteOptions.swift

```diff
@@ -303,6 +339,10 @@ open class RouteOptions: NSObject, NSSecureCoding {
      */
     open var includesExitRoundaboutManeuver = false
     
+    open var includeVoiceInstructions = false
+    
+    open var unitsForVoiceInstructions: VoiceUnit = .imperial
```

Comment:
Since this property takes only one value, the plural “units” is misleading. Assuming the type is renamed to `MeasurementSystem`, this property should be called `lengthMeasurementSystem`.